docs/learn/prediction-markets.md

```diff
@@ -236,21 +236,18 @@ popular vote (cast between the two parties) are approximately 51.11% and 48.89%
 
 <!-- TODO Shouldn't this go into the futarchy section? -->
 
-Prediction markets can be traced back to writings of [Ludwig von Mises][mises]
-and [Frederik Hayek][hayek], however it is the economist Robin Hansen that is
-perhaps the best-known proponent of them today. Hansen's writings have a very
-significant theoretic implication on how to implement prediction markets in a
-blockchain setting. He posits in _Shall We Vote on Values, But Bet on
-Beliefs?_[^1] that the primary problem that can be solved is that of _info
-problems_, that is the difficulty of aggregating information among many
-individuals with all different views of the subject.
+Prediction markets can be traced back to writings of
+[Ludwig von Mises](https://cdn.mises.org/Economic%20Calculation%20in%20the%20Socialist%20Commonwealth_Vol_2_3.pdf)
+and [Frederik Hayek](https://www.kysq.org/docs/Hayek_45.pdf), however it is the
+economist Robin Hansen that is perhaps the best-known proponent of them today.
+Hansen's writings have a very significant theoretic implication on how to
+implement prediction markets in a blockchain setting. He posits in _Shall We
+Vote on Values, But Bet on Beliefs?_[^1] that the primary problem that can be
+solved is that of _info problems_, that is the difficulty of aggregating
+information among many individuals with all different views of the subject.
```

Comment:
This change removes the hyperlink abstractions `[mises]` and `[hayek]`. Why did this change happen?

Also, when I copy the version of `prediction-markets.md` from `main` into you fork and run `yarn fmt`, these changes do not occur, so I don't see how this is something that `prettier` could have done.

Reply:
Because the Crowdin can't recognize such hyperlinks, but after modification there is no problem

Reply:
How were these changes applied? Manually, or using a formatter?